src/assets/css/main.css

```diff
@@ -140,8 +140,8 @@
 }
 
 .icon-emoji-admin {
-  max-width: 24px; 
-  max-height: 24px; 
+  max-width: 24px;
+  max-height: 24px;
```

Comment:
Thanks. We should alter prettier / to make sure it works for css files too, I'll add that now.